ddpui/tests/services/test_org_cleanup_service.py

```
@@ -235,3 +238,450 @@ def test_delete_orchestrate_pipelines_none(mock_prefect_service):
+        mock_secrets,
+        mock_airbyte,
+        mock_prefect,
+    ):
+        org = Org.objects.create(
+            name="OrgFullDry",
+            airbyte_workspace_id="ws-full-dry",
+            slug="org-full-dry",
+        )
+        mock_airbyte.get_sources.return_value = {"sources": []}
+        mock_airbyte.get_destinations.return_value = {"destinations": []}
+        mock_dbt_manager.get_org_dir.return_value = "/tmp/fake_org_dir"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake_org_dir"

(S108)